src/ssl.c

```diff
@@ -45218,10 +45254,13 @@ int wolfSSL_CTX_use_PrivateKey(WOLFSSL_CTX *ctx, WOLFSSL_EVP_PKEY *pkey)
 
 #endif /* OPENSSL_EXTRA */
 
+#if defined(HAVE_EX_DATA) || defined(FORTRESS) || defined(WOLFSSL_WPAS_SMALL)
+/*
```

Comment:
This is disabling a section of macros. Should these be removed instead?

Reply:
yeah, meant to before committing.  done.